routes/index.js

```diff
@@ -42,11 +42,11 @@ router.get('/', function(req, res) {
             });
             ++i;
           }
-          
+
           res.render('index', {title: 'VibeCheck',
```

Comment:
Could we just grab a random poll instead, so that we dont have to sort all by popular just to get one document?

Reply:
Yes, how do we do that?